components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -214,3 +214,9 @@
     transform: rotate(180deg);
   }
 }
+
+.primary-nav__link--level-0:last-child {
```

Comment:
This doesn't work because it's affecting every `.primary-nav__link--level-0` since they're all the "last-child" of their respective `<li>` elements. (see screenshot) We'll need to target the last `.primary-nav__item--level-0` and then apply these styles to the link inside that.

![Screen Shot 2022-11-03 at 4 35 34 PM](https://user-images.githubusercontent.com/1663810/199839164-cd5aa1e8-f546-4559-93d0-aa488fe833d1.png)

It doesn't affect anything that has the class `.primary-nav__toggle--level-0` which is everything that has child links, so it's not super obvious.

Also, we need to remove the `padding-left` from the first item, to make it align as well.

There is a mixin at the top of that file `@mixin primary-nav-item-level-0` that is implemented in both `.primary-nav__link--level-0` and `.primary-nav__toggle-level-0` where shared styles should go. I think that's going to be the right place to put this css.

Inside the media query on line 31, I think I'd add this, which should make it work the way we expect.

```
.primary-nav__item--level-0:first-child & {
  padding-left: 0;
}

.primary-nav__item--level-0:last-child & {
  padding-right: 0;
}
```